crb.c

```diff
@@ -298,8 +274,34 @@ size_t crb_send(struct tpmbuff *buf)
 	return buf->len;
 }
 
-size_t crb_recv(struct tpmbuff *buf)
+size_t crb_recv(__attribute__((unused)) enum tpm_family, __attribute__((unused)) struct tpmbuff *buf)
 {
 	/* noop, currently send waits until execution is complete*/
```

Comment:
Not sure what this comment means. Is there no such thing as a CRB receive operation?

Reply:
Not really, you just read the CRB buffer back and specifically in the case of an extend the return is less than the CRB buffer so its more about just waiting for the TPM state to change.